dataland-qa-service/src/main/kotlin/org/dataland/datalandqaservice/api/QaApi.kt

```diff
@@ -37,10 +39,38 @@ interface QaApi {
     @PreAuthorize("hasRole('ROLE_REVIEWER')")
     fun getUnreviewedDatasetsIds(): ResponseEntity<List<String>>
 
+    /**
+     * A method to get the QA review status of an uploaded dataset for a given identifier
+     * @param identifier the identifier
+     */
+    @Operation(
+        summary = "Gets the QA review status of an uploaded dataset for a given identifier.",
+        description = "Gets the QA review status of an uploaded dataset for a given identifier.",
+    )
+    @ApiResponses(
+        value = [
+            ApiResponse(responseCode = "200", description = "Found a QA review status corresponding the identifier."),
+            ApiResponse(responseCode = "404", description = "Found no QA review status corresponding the identifier."),
+        ],
+    )
+    @GetMapping(
+        value = ["/datasets/{dataId}"],
+        produces = ["application/json"],
+    )
+    @PreAuthorize(
+        "hasRole('ROLE_REVIEWER') " +
+            "or hasRole('ROLE_ADMIN') " +
+            "or @SecurityUtilsService.isUserAskingQaReviewStatusOfUploadedDataset(#dataId)",
```

Comment:
Der Name der Funktion isUserAskingQaReviewStatusOfUploadedDatasets vielleicht ändern zu isUserAskingQaReviewStatusOfOwnDatasets, da die Funktion ja prüft ob der User selber die Daten hochgeladen hat.
Eine Frage die abseits dazu bei mir aufkam ist die, wie das mit den anderen Company-Roles funktioniert? Sollen alle Mitglieder einer Company den Status von Datensätzen der Company zu der sie gehören abfragen können?

Reply:
renamed.
Gute Frage, alle company roles ist vielleicht übertrieben, aber man könnte über zusätzliche rechte für die company roles uploader and owner nachdenken. (würde ein neues clientsecret zur communication mit dem community service erfordern). 
Ich würde es aber eher in einen neuen ticket sehen?

Reply:
Würde ich auch in einem eigenen Ticket sehen